site/src/components/data/financial.md

```diff
@@ -18,11 +23,28 @@ The following example:
 
 Generates the following:
 
-{{{ dynamic-include 'codepen' html="financial-example-html" js="financial-example-js" }}}
-
 {{{financial-example-html}}}
 <script type="text/javascript">
 {{{financial-example-js}}}
-</script>
+</script> 
+
+{{{ dynamic-include 'codepen' html="financial-example-html" js="financial-example-js" }}}
+
+## Stream
+The component exposes a stateful streaming interface.
+You can see this in action in the [streaming chart example](../../examples/streaming/index.html).
+First create a stream from a random.financial instance with `financial.stream()`.
```

Comment:
I think this is a bit confusing, maybe reword it to -

First create a stream from an instance of ```fc.data.random.financial``` by calling the ```stream()``` method.